routes/postsRoute.js

```diff
@@ -98,13 +89,10 @@ router.get("/", async (req, res) => {
 });
 
 router.get("/:id", async (req, res) => {
-  const id = req.params.id;
   try {
-    const currentPost = await Post.query()
-      .findById(id)
-      .withGraphFetched("user")
-      .withGraphFetched("comments")
-      .withGraphFetched("comments.user");
+    const id = req.params.id;
+    const currentPost = getPostById(id, withComment);
```

Comment:
you didn't define the withComment variable here in this route

Reply:
We are gonna get it from query parameters, why define it?

Reply:
but you didn't get it from the query params
  const withComment = req.query.withComment;